src/components/fieldset/index.md.njk

```diff
@@ -25,6 +25,10 @@ If you’re using the examples or macros for [radios](../radios), [checkboxes](.
 
 The first element inside a fieldset must be a `legend` which describes the group of inputs. This could be a question, such as ‘What is your current address?’ or a statement like ‘Personal details’.
 
+If you’re asking just [one question per page](../../patterns/question-pages/#start-by-asking-one-question-per-page) as recommended, you can set the contents of the `<legend>` as the page heading, as shown in the example below.
```

Comment:
Is it a conscious choice that this page doesn't get the sentence "This is good practice as it means that users of screen readers will only hear the contents once."?

Reply:
No it isn't. It wasn't a conscious choice to make the wording so similar across all the pages, it just evolved that way and this was one of the first ones I wrote.

We can add it in to make it consistent.